src/components/SnazzyButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { CSSProperties } from 'react';
+import { Button, ButtonProps } from '@patternfly/react-core';
+import { ColoredTd } from './ColoredTd';
+
+type SnazzyButtonProps = {
+  color: string;
+  isSnazzy: boolean;
+  customStyle?: string;
+} & ButtonProps;
+
+const SnazzyButton: React.FC<SnazzyButtonProps> = ({ color, isSnazzy, customStyle, ...rest }) => {
+  if (isSnazzy) {
+    return (
+      <ColoredTd color={color}>
+        <Button style={customStyle as CSSProperties} {...rest} />
+      </ColoredTd>
+    );
+  }
+
+  return <Button style={customStyle as CSSProperties} {...rest} />;
```

Comment:
when isSnazzy is false (when this code runes) we don't want to apply the style (or class after above changes) we just want this to return the normal button.